server/src/fixtures/README.md

```
@@ -1,9 +1,9 @@
-# fixtures/ — Fixture Store, Color Pipeline & Movement
+# fixtures/ -- Fixture Store, Color Pipeline & Movement
 
-## Fixture Store
-- `fixture-store.ts`: CRUD for `FixtureConfig[]`, persisted to `./config/fixtures.json`
-- `group-store.ts`: fixture groups with membership tracking, persisted to `./config/groups.json`
-- `user-fixture-store.ts`: user-created fixture templates (custom definitions)
+## Stores
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)